ipatests/test_webui/test_pwpolicy.py

```diff
@@ -54,4 +51,214 @@ def test_crud(self):
+            self.modify_policy(field, minimum_value_expected_error,
+                               minimum_value)
+        self.navigate_to_entity(pwpolicy.ENTITY)
+        self.delete_record(pwpolicy.group.PKEY)
+
+        # Negative test for policy priority
+        field_priority = 'cospriority'
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA7, dialog_btn=None)
+        # non integer for policy priority
+        self.fill_input(field_priority, non_integer)
+        self.assert_field_validation(non_interger_expected_error,
+                                     field=field_priority)
+        #  lower bound of data range
+        self.fill_input(field_priority, minimum_value)
+        self.assert_field_validation(minimum_value_expected_error,
+                                     field=field_priority)
+        # upper bound of data range
+        self.fill_input(field_priority, maximum_value)
+        self.assert_field_validation(expect_error='Maximum value is'
+                                                  ' 2147483647',
+                                     field=field_priority)
+        self.close_all_dialogs()
+
+    def modify_policy(self, pwdfield, expected_error, value):
+        """
+        modifying password policy values and asserting expected error
+        """
+        self.fill_textbox(pwdfield, value)
+        self.wait_for_request()
+        self.assert_field_validation(expected_error, field=pwdfield)
+        self.facet_button_click('revert')
+
+    @screenshot
+    def test_undo_refresh_revert(self):
+        """
+        Test to verify undo/refresh/revert
+        """
+        self.init_app()
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA_RESET)
+        self.navigate_to_record(pwpolicy.PKEY6)
+
+        # test undo
+        self.fill_fields(pwpolicy.DATA_RESET['mod'])
+        mod_field = 0
+        for field in FIELDS:
+            modified_value = (pwpolicy.DATA_RESET['mod'][mod_field][2])
+            self.click_undo_button(field)
+            self.verify_btn_action(field, modified_value)
+            mod_field + 1
+
+        # test refresh
+        mod_field = 0
+        for field in FIELDS:
+            self.fill_fields(pwpolicy.DATA_RESET['mod'])
+            modified_value = (pwpolicy.DATA_RESET['mod'][mod_field][2])
+            self.facet_button_click('refresh')
+            self.verify_btn_action(field, modified_value)
+            mod_field + 1
+
+        # test revert
+        mod_field = 0
+        for field in FIELDS:
+            self.fill_fields(pwpolicy.DATA_RESET['mod'])
+            modified_value = (pwpolicy.DATA_RESET['mod'][mod_field][2])
+            self.facet_button_click('revert')
+            self.verify_btn_action(field, modified_value)
+            mod_field + 1
+
+        # cleanup
+        self.navigate_to_entity(pwpolicy.ENTITY)
+        self.delete_record(pwpolicy.group.PKEY6)
+
+    def verify_btn_action(self, field, mod_value, negative=True):
```

Comment:
This method has a specific purpose: check that value of field is equal or not equal to some provided value. But the name is quite generic, not describing what it does. So it could easily by `field_equals` or something like that.